db/models.py

```diff
@@ -45,3 +55,65 @@ class MovieSession(models.Model):
 
     def __str__(self):
         return f"{self.movie.title} {str(self.show_time)}"
+
+
+class Order(models.Model):
+    created_at = models.DateTimeField(auto_now_add=True)
+    user = models.ForeignKey(
+        settings.AUTH_USER_MODEL,
+        on_delete=models.CASCADE
+    )
+
+    def __str__(self):
+        return f"{self.created_at}"
+
+    class Meta:
+        ordering = ["-id"]
+
+
+class Ticket(models.Model):
+    row = models.IntegerField()
+    seat = models.IntegerField()
+    movie_session = models.ForeignKey("MovieSession", on_delete=models.CASCADE)
+    order = models.ForeignKey("Order", on_delete=models.SET_NULL, null=True)
+
+    def __str__(self):
+        time_format = "%Y-%m-%d %H:%M:%S"
+        return f"{self.movie_session.movie.title} " \
+               f"{self.movie_session.show_time.strftime(time_format)} " \
```

Comment:
You already defined __str__ method for movie session so its may be just f"{self.movie_session} ...